btas/generic/converge_class.h

```diff
@@ -354,5 +358,174 @@ namespace btas {
       return sqrt(abs(nrm));
     }
   };
+
+  template<typename Tensor>
```

Comment:
need dox

Reply:
This isn't necessary. I thought it was removed, doing that now.